### ISSUE TITLE
OMV Virtualbox Dashboard fails if Webdav Max Upload not set

### ISSUE THREAD

**Issue by andibing** [OpenMediaVault-Plugin-Developers/openmediavault-webdav#9]
OMV Version : 3.0.81

When attempting to start VirtualBox plugin it fails.

It would appear the error is caused by the line:
   "client_max_body_size M;"
in this file:  /etc/nginx/openmediavault-webgui.d/openmediavault-webdav.conf
(Note the missing size)

Since this is created by the file /usr/share/openmediavault/mkconf/nginx.d/90-webdav
   "client_max_body_size ${upload_max}M;"
Which seems to fail if "upload_max" is not set.

**Workaround by enabling WebDav and configuring the maxiumum upload. Can a default be applied?**

**Comment by ryecoaaron** [comment]
The web interface has a default of 2 and minimum of 1.  It shouldn't be possible to leave it unset from the web interface.  Did you manually change something?

**Comment by andibing** [comment]
Not changed any of these specific settings (WebDAV, nginx etc)...  but yes the box has been fairly customised, and upgraded through several versions.... so definitely not a clean install! :-O

However, I did notice that attempting to start nginx after the upgrade to 3.0.x caused the same error. I thus left /Services/nginx disabled in the WebUI.

Only after a more recent reboot did the Virtualbox WebUI stop.

For background...
In the WebUI, Services->WebDAV, the "Filesize upload limit (MiB)" was blank... with no defaults.
Thought I saw 25M mentioned in a config file or example so setting it to that in the WebUI resolved the problem. :-)
Making WebDAV work, meant that the VirtualBox (phpvirtualbox) settings worked.


Feel free to close the ticket... problem solved. But wanted to document in case it correlates with something else.
